site/content/en/guides/_index.md

```diff
@@ -7,5 +7,5 @@ menu:
   main:
     weight: 20
 description: >
-    Reference for Kustomize CLI commands
+    Reference for Kustomize usage and best practices
```

Comment:
@monopole Can we call this *best practices*?

Reply:
yes, in the context of kustomize.
it goes here
https://kubernetes-sigs.github.io/kustomize/guides/

@dearchap please consider providing larger prs to improve the docs, thanks!